ccm_web/server/src/health/health.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Controller, Get } from '@nestjs/common'
+import { ConfigService } from '@nestjs/config'
+import {
+  HealthCheck, HealthCheckResult, HealthCheckService, HttpHealthIndicator, SequelizeHealthIndicator
+} from '@nestjs/terminus'
+
+import { Config } from '../config'
+
+@Controller('health')
+export class HealthController {
+  constructor (
+    private readonly configService: ConfigService<Config, true>,
+    private readonly health: HealthCheckService,
+    private readonly http: HttpHealthIndicator,
+    private readonly db: SequelizeHealthIndicator
+  ) {}
+
+  @Get()
+  @HealthCheck()
+  async check (): Promise<HealthCheckResult> {
+    const appDomain = this.configService.get('server.domain', { infer: true })
+    return await this.health.check([
+      async () => await this.http.pingCheck('root', `https://${appDomain}`),
+      async () => await this.db.pingCheck('database', { timeout: 3000 })
```

Comment:
I don't think the DB connection is established to make this ping work. The DB connection needs to be give to  make that check. So you usually do it at `health.module.ts` but as of now DB connection is not established. 
More something like this should be done https://github.com/nestjs/terminus/blob/master/sample/007-sequelize-app/src/health/health.module.ts

Reply:
@pushyamig, I'm not sure about that. Based on my understanding of the docs on the `SequelizeModule` (https://docs.nestjs.com/techniques/database#sequelize-integration), you shouldn't need to import the `SequelizeModule` in other modules if it's in the `AppModule`.

Furthermore, I am seeing SQL emitted from `sequelize` (`SELECT 1`) when I hit this endpoint.

Reply:
Yeah, I saw this too. I'm not sure what to make of it, or how else to test the database being down.

Reply:
@jonespm The test I did is a valid way of looking if DB up/down?